backend/src/main/java/com/google/models/Alert.java

```diff
@@ -32,14 +32,18 @@
     UNRESOLVED
   };
 
+  private static final long DEFAULT_ID = 0L;
```

Comment:
just checking, the id won't ever be 0 right

Reply:
yes, once an entity is stored in to the datastore, the ID assigned will not be zero. It will be an 8 digit long.

Reply:
So from my understanding, `Alert createAlertFromEntity(Entity alertEntity)` will only return id = 0 if the Entity. is not yet stored to the datastore (or if entity unsuccessfully stored in datastore ie due to some backend/API call error), but otherwise will return the 8 digit long?

Reply:
yes, that is correct!